debian/libopx-pas-dev.install

```diff
@@ -0,0 +1,2 @@
+usr/include/opx/dn_platform_utils.h
+usr/lib/*/*.so*
```

Comment:
This seems to be what's causing Smoketest to fail. Does the -dev package need to include the shared object? -dev packages in other repos seem to just install headers. Ex:
https://github.com/open-switch/opx-nas-l2/blob/master/debian/libopx-nas-l2-dev.install
https://github.com/open-switch/opx-sdi-sys/blob/master/debian/libopx-sdi-sys-dev.install
Although base-model is an exception -- base-model-dev includes static libraries (*.a) and base-model includes the shared objects (*.so).